back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -33,6 +33,21 @@ def reorder
     end
   end
 
+  # For use by homepage to get status counts of visible top-level (not in folder) admin_publications,
+  # using: .../web_api/v1/admin_publications/status_counts?remove_not_allowed_parents=true&depth=0
+  def status_counts
+    publication_filterer = AdminPublicationsFilteringService.new
+    publications = policy_scope(AdminPublication.includes(:parent))
+    publications = publication_filterer.filter(publications, params)
+
+    # For non-admins: publication_status: :draft count should always be zero, and thus not included in response
+    counts = publications.group(:publication_status).count
+
+    authorize :admin_publication, :status_counts
```

Comment:
Moving the authorization to the top would be more efficient if access is denied.